src/renderer/components/watch-video-playlist/watch-video-playlist.js

```diff
@@ -129,6 +129,15 @@ export default defineComponent({
         nextTick(() => this.scrollToCurrentVideo())
       }
     },
+    playlistId: function (newVal, oldVal) {
+      if (oldVal !== newVal) {
```

Comment:
I don't understand this conditional...

Reply:
We don't want to refetch the playlist if the id is the same

Reply:
It's called when value not updated too? (It's inside `watch`)

Reply:
I'm not sure, we definitely set the property every time the video changes, but i'm not sure if watch is triggered on every set or just when the value changes. We have this pattern inside watches in a few places which made me think it might get triggered for every set  but i'm not sure.

Reply:
I can always remove the check later, if we find out that it's redundant.